dockers/docker-fpm-frr/docker_init.sh

```diff
@@ -56,41 +56,32 @@ if [[ ! -z "$NAMESPACE_ID" ]]; then
    update_default_gw 6
 fi
 
-if [ -z "$CONFIG_TYPE" ] || [ "$CONFIG_TYPE" == "separated" ]; then
-    CFGGEN_PARAMS=" \
-        -d \
-        -y /etc/sonic/constants.yml \
-        -t /usr/share/sonic/templates/bgpd/gen_bgpd.conf.j2,/etc/frr/bgpd.conf \
-        -t /usr/share/sonic/templates/zebra/zebra.conf.j2,/etc/frr/zebra.conf \
-        -t /usr/share/sonic/templates/staticd/gen_staticd.conf.j2,/etc/frr/staticd.conf \
-    "
-    MGMT_FRAMEWORK_CONFIG=$(echo $FRR_VARS | jq -r '.frr_mgmt_framework_config')
-    if [ -n "$MGMT_FRAMEWORK_CONFIG" ] && [ "$MGMT_FRAMEWORK_CONFIG" != "false" ]; then
-        CFGGEN_PARAMS="$CFGGEN_PARAMS \
-            -t /usr/local/sonic/frrcfgd/bfdd.conf.j2,/etc/frr/bfdd.conf \
-            -t /usr/local/sonic/frrcfgd/ospfd.conf.j2,/etc/frr/ospfd.conf \
-        "
-    else
-        rm -f /etc/frr/bfdd.conf /etc/frr/ospfd.conf
-    fi
-    sonic-cfggen $CFGGEN_PARAMS
-    echo "no service integrated-vtysh-config" > /etc/frr/vtysh.conf
-    rm -f /etc/frr/frr.conf
-elif [ "$CONFIG_TYPE" == "split" ]; then
+if [ "$CONFIG_TYPE" == "split" ]; then
     echo "no service integrated-vtysh-config" > /etc/frr/vtysh.conf
     rm -f /etc/frr/frr.conf
     write_default_zebra_config /etc/frr/zebra.conf
 elif [ "$CONFIG_TYPE" == "split-unified" ]; then
     echo "service integrated-vtysh-config" > /etc/frr/vtysh.conf
     rm -f /etc/frr/bgpd.conf /etc/frr/zebra.conf /etc/frr/staticd.conf
     write_default_zebra_config /etc/frr/frr.conf
-elif [ "$CONFIG_TYPE" == "unified" ]; then
-    CFGGEN_PARAMS=" \
-        -d \
-        -y /etc/sonic/constants.yml \
-        -T /usr/local/sonic/frrcfgd \
-        -t /usr/share/sonic/templates/gen_frr.conf.j2,/etc/frr/frr.conf \
-    "
+elif [ "$CONFIG_TYPE" == "separated" ]; then
+    echo "Config Type separated is not supported"
```

Comment:
>separated is not supported [](http://example.com/codeflow?start=22&length=26)

This breaking change will break warm-reboot, because the old image config_db.json will be carried over to new image. Please update config in src/sonic-utilities/scripts/db_migrator.py, to use a supported "docker_routing_config_mode"

Reply:
To solve this, db_migrator changes are added 
https://github.com/sonic-net/sonic-utilities/pull/3840